details/structured-groups.md

```diff
@@ -0,0 +1,85 @@
+# Structured groups
+
+Group ID represents a group/roles concept in the ROR ACL. You can define a neat, human-readable ID of the group when you introduce groups on the configuration level.
+However, when group IDs come from external integrations, the group IDs may have an unfriendly format.
+That's why we introduced a structured group concept, which consists of:
+* Group ID - a group identifier, used in the `groups`, `groups_or`, and `groups_and` ACL rules
```

Comment:
we should add that ROR operates only on this (the variables return IDs not, names).